Update service_remapping_rules.md (#36904)
DataDog/documentation: content/en/tracing/services/service_remapping_rules.md

- **Historical data:** Changes made by remapping rules affect only telemetry ingested while a rule is active, and past data is not updated retroactively. Deleting or modifying a rule stops it from applying to new data, but does not revert names on previously ingested data.
- **Logs service remapper:** Service remapping rules occur before logs pipelines. If the logs service remapper and remapping rules are both applied to a service, the remapping rules take precedence. 
- **Dashboards and monitors:** Existing queries that reference old service names are not automatically updated. Review and update these manually.

> **Integration and custom overrides:** If integration overrides or custom overrides fall within the scope of a remapping rule, they will also be remapped. [Remove integration overrides][15] for the best APM experience.

Reviewer: ```suggestion
**Integration and custom overrides:** If integration overrides or custom overrides fall within the scope of a remapping rule, they are also remapped. [Remove integration overrides][15] for the best APM experience.
```
